CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.9.1 (2021-06-14)
+
+- Added support for `marked@2.0.7`. [#163](https://github.com/blackbaud/skyux-docs-tools/pull/163)
```

Comment:
```suggestion
- Added support for `marked@2.0.7` and updated tests to be screen size agnostic. [#163](https://github.com/blackbaud/skyux-docs-tools/pull/163)
```

Reply:
@blackbaud-paulah We don't normally call out changes to our internal tests in our CHANGELOG as it doesn't affect the consumer of the library. Thoughts?

Reply:
Cool. I didn't know that. Thanks for explaining.